resources/views/courses/details.blade.php

```diff
@@ -1,27 +1,30 @@
-
 @extends('layouts.app')
 
-@section('title', 'Course details')
+@section('title', $course->name)
 
 @section('content')
 <div class="padded content">
-	<h1>{{ "HERE GOES THE COURSE NAME" }}</h1>
+	<h1>{{ $course->name }}</h1>
 	<div>
-		<p class="coursekey">Your course key is <br> <code>{{ isset($courseKey) ? $courseKey : "HELLO" }}</code></p>
+		<p class="coursekey">Your course key is <br> <code>{{ isset($course->course_key) ? $course->course_key : "ERROR" }}</code></p>
 		<h2>Overview:</h2>
-		<p><strong>Course name:</strong> Cool course name</p>
+		<p><strong>Course name:</strong> {{ $course->name }}</p>
 		<p><strong>Teacher(s):</strong></p>
-		<ul>
-			<li>Mr. Monday</li>
-		</ul>
-		<p><strong>Course semester:</strong> Fall 2018</p>
-		<p><strong>Course type:</strong> Client</p>
-		<p><strong>Course schedule:</strong> Monday, Thursday</p>
-		<p><strong>Starting time:</strong> 10:30 AM</p>
-		<p><strong>Max. team size:</strong> 4</p>
-		<p><strong>Associated courses:</strong></p>
-		<ul>
-		</ul>
+		{{-- <ul>
+			@foreach ($course->$teachers as $teacher)
+			<li>{{ $teacher->name }}</li>
+			@endforeach
+		</ul> --}}
+		<p><strong>Course schedule:</strong> {{ $course->schedule }}</p>
+		<p><strong>Max. team size:</strong> {{ $course->max_team_size }}</p>
```

Comment:
We're moving away from specifying a maximum team size. Since this is related to #107 could we do the changes here? 😄